channelacceptor/chainedacceptor.go

```diff
@@ -0,0 +1,36 @@
+package channelacceptor
```

Comment:
Would be nice to have some basic coverage tests for this package now that it no longer lives in the main package.